MAUI/StepProgressBar/description.md

```diff
@@ -0,0 +1,413 @@
+                    x:Name="stepProgress"
+                    VerticalOptions="Center"
+                    HorizontalOptions="Center"                                        
+                    Orientation="Horizontal"                                                                                       
+                    LabelSpacing="12"
+                    LabelPosition="Bottom"
+                    ActiveStepIndex="2"
+                    ActiveStepProgressValue="50"    
+                    ItemsSource="{Binding StepProgressItem}">
+</stepProgressBar:SfStepProgressBar>                                                                                              
+
+<ContentPage.BindingContext>
+    <local:ViewModel />
+</ContentPage.BindingContext>
+
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="9" %}
+
+ViewModel viewModel = new ViewModel();
+SfStepProgressBar stepProgressBar = new SfStepProgressBar()
+{
+    VerticalOptions = LayoutOptions.Center,
+    HorizontalOptions = LayoutOptions.Center,
+    Orientation = StepProgressBarOrientation.Horizontal,
+    LabelSpacing = 12,
+    LabelPosition = LabelPosition.Bottom,
+    ActiveStepIndex = 2,
+    ActiveStepProgressValue = 50,
+    ItemsSource = viewModel.StepProgressItem,
+};
+
+this.Content = stepProgressBar;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customization using Font icon
+
+You can add the images with font icons to the [ImageSource]() property in `SfStepProgressBar`. The following code example explains how to set an image in step view.
+
+#### Initialize view model
+
+* Create a simple Observable Collection of item with font icons for the step progressbar,
+
+{% tabs %}
+{% highlight c# tabtitle="ViewModel.cs" hl_lines="26 27 28" %}
+
+ public class ViewModel
+ {
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    private ObservableCollection<StepProgressBarItem> stepProgressItem;
+
+    /// <summary>
+    /// The Step progress bar item collection.
+    /// </summary>
+    public ObservableCollection<StepProgressBarItem> StepProgressItem
+    {
+        get
+        {
+            return stepProgressItem;
+        }
+        set
+        {
+            stepProgressItem = value;
+        }
+    }
+
+    public ViewModel()
+    {
+        stepProgressItem = new ObservableCollection<StepProgressBarItem>();
+        stepProgressItem.Add(new StepProgressBarItem() { ImageSource = new FontImageSource() { Glyph = "\ue760", Size = 32, FontFamily = "IconFont", Color = Colors.White } });
+        stepProgressItem.Add(new StepProgressBarItem() { ImageSource = new FontImageSource() { Glyph = "\ue77f", Size = 32, FontFamily = "IconFont", Color = Colors.White } });
+        stepProgressItem.Add(new StepProgressBarItem() { ImageSource = new FontImageSource() { Glyph = "\ue786", Size = 32, FontFamily = "IconFont", Color = Colors.White } });
+    }
+ }
+
+{% endhighlight %}
+{% endtabs %}
+
+* Bind the Collection to step progressbar
```

Comment:
* Bind the Collection to step progressbar.

Reply:
Content modified